orchestrator/cli/generator/generator/product.py

```diff
@@ -45,20 +40,20 @@ def generate_product(context: dict) -> None:
     fixed_inputs = config.get("fixed_inputs", [])
     product_blocks = config.get("product_blocks", [])
 
-    non_standard_fixed_inputs = get_non_standard_fixed_inputs(fixed_inputs)
-    int_enums = get_int_enum_fixed_inputs(fixed_inputs)
-    str_enums = get_str_enum_fixed_inputs(fixed_inputs)
+    non_standard_fixed_inputs = get_non_standard_fields(fixed_inputs)
+    int_enums = get_int_enums(fixed_inputs)
+    str_enums = get_str_enums(fixed_inputs)
```

Comment:
I think it would be cleaner to have a function that handles both. I see this done twice, here and in product_block.

Reply:
But I do need two separate lists, one for the int_enums and one for the str_enums. When I combine both into one than the new function should either have an argument to switch between int an str, or should return both lists at once, both solutions do make it cleaner in my opinion.